src/nupic/encoders/SparsePassThroughEncoder.hpp

```diff
@@ -0,0 +1,114 @@
+/* ---------------------------------------------------------------------
+* Numenta Platform for Intelligent Computing (NuPIC)
+* Copyright (C) 2014-2016, Numenta, In  Unless you have an agreement
+* with Numenta, In, for a separate license for this software code, the
+* following terms and conditions apply:
+*
+* This program is free software: you can redistribute it and/or modify
+* it under the terms of the GNU Affero Public License version 3 as
+* published by the Free Software Foundation.
+*
+* This program is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+* See the GNU Affero Public License for more details.
+*
+* You should have received a copy of the GNU Affero Public License
+* along with this program.  If not, see http://www.gnu.org/licenses.
+*
+* http://numenta.org/licenses/
+* ---------------------------------------------------------------------
+*/
+
+#ifndef NTA_sparsepassthroughencoder_HPP
+#define NTA_sparsepassthroughencoder_HPP
+
+#include <iostream>
+#include <string> 
+#include <vector>
+#include <tuple>
+#include <map> 
+
+#include <nupic/encoders/PassThroughEncoder.hpp>
+
+using namespace std;
+using nupic::UInt;
+
+namespace encoders {
+
+	/**
+	* Sparse Pass Through Encoder
```

Comment:
This encoder can be replaced by the SDR.

Reply:
Good for SDR! Removing..